source/ngf-mtl/impl.mm

```diff
@@ -1831,12 +1901,15 @@ ngf_error ngf_cmd_begin_render_pass(
             clear_info->clear_color[2],
             clear_info->clear_color[3]);
       }
-      mtl_desc.resolveTexture =
-          rt->is_default ? CURRENT_CONTEXT->frame.resolve_attachment_texture() : nil;
-      if (mtl_desc.resolveTexture) {
+
+      if (attachment_desc.sample_count > NGF_SAMPLE_COUNT_1) {
```

Comment:
if there are no resolve attachments specified at all, we should
1) _not_ set `resolveTexture`
2) respect the user-specified storeAction